src/Files.App/Filesystem/Archive/ArchiveCreator.cs

```diff
@@ -118,6 +130,16 @@ public async Task<bool> RunCreationAsync()
 			}
 		}
 
-		private void Compressor_Compressing(object? _, ProgressEventArgs e) => Progress.Report(e.PercentDone);
+        private void Compressor_CompressionFinished(object? sender, EventArgs e)
+        {
+			fsProgress.Percentage = null;
+            fsProgress.ReportStatus(Shared.Enums.FileSystemStatusCode.Success);
+        }
+
+        private void Compressor_Compressing(object? _, ProgressEventArgs e)
+		{
+			fsProgress.Percentage = e.PercentDone;
+			fsProgress.Report();
+        }
 	}
```

Comment:
The overall formatting of this file is broken. Could you make pass VS formatter on it while it's being modified?

Reply:
It was broken due to the difference of spaces and tabs. My Visual Studio keeps replacing tabs to spaces wherever I have edited.